app/controllers/users_controller.rb

```diff
@@ -1,4 +1,4 @@
-class UsersController < ApplicationController
+class UsersController < ApplicationController # rubocop:disable Metrics/ClassLength
```

Comment:
No way to shorten?

Reply:
Not in a way that makes it more readable I don't think